frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.scss

```diff
@@ -0,0 +1,91 @@
+/* top info styles */
+.nb-create-bc-info {
+    background-color: var(--pf-global--BackgroundColor--300);
+    color:var(--pf-global--Color--dark-200);
+    margin-top: var(--pf-global--spacer--sm);
+    margin-bottom: var(--pf-global--spacer--sm);
+    padding: var(--pf-global--spacer--md);
+    width:100%;
+}
+
+.nb-create-bc-info-header {
+    display: inline-flex;
+    width:100%;
+}
+
+.nb-create-bc-info-header > h2 {
+    display: inline-flex;
+}
+
+.nb-create-bc-info-header__close {
+    color:var(--pf-global--Color--dark-100);
+    margin-left: auto;
+    text-align: right;
+    font-size: var(--pf-global-FontSize--lg);
+}
+
+.nb-create-bc-info-body {
+    display:block;
+    margin-bottom: var(--pf-global--spacer--lg);
+}
+
+.nb-create-bc-info-body__description {
+    color: var(--pf-global--Color--dark-200);
+    display: block;
+    margin-top: var(--pf-global--spacer--md);
+    margin-bottom: var(--pf-global--spacer--md);
+}
+
+/* Step pages */
+.nb-create-bc-step-page {
+    display: block;
+    width: 100%;
+}
+
+.nb-bc-bs-table__data {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.nb-create-bc-step-page-form {
+    display: block;
+    width:80%;
+    margin-right: auto;
+    margin-bottom:var(--pf-global--spacer--lg);
+}
+
+.nb-create-bc-step-page-form__element {
+    margin-top:var(--pf-global--spacer--md);
+    margin-bottom:var(--pf-global--spacer--md);
+}
+
+.nb-create-bc-step-page-form__element--light-text {
+    color:var(--pf-global--Color--400);
+}
+
+.nb-bc-step-page-form__element--short {
+    width:30%;
+    display:flex;
+    margin-top: var(--pf-global--spacer--md);
+}
+
+.nb-bc-bs-page__table--short {
+    width:90%;
+    max-height: 50vh;
```

Comment:
Another viewport question :) Has this been considered for small screens? It seems a bit odd to hide content like this, but I am only looking at screenshots so forgive me if I'm missing something.